cumulusci/cli/cci.py

```diff
@@ -1053,6 +1053,41 @@ def org_list(runtime, plain):
     persistent_table.echo(plain)
 
 
+@org.command(name="prune", help="Removes all expired orgs from the current project")
+@pass_runtime(require_project=False, require_keychain=True)
```

Comment:
```suggestion
@pass_runtime(require_project=True, require_keychain=True)
```

to make sure CCI raises an error proactively if you try using this command outside a cci project folder, rather than a more obscure error later when something tries to use project_config but it's None